src/pages/CamerasPage/Footer/Tube/Kenos/Kenos.tsx

```diff
@@ -0,0 +1,12 @@
+import styles from "./Kenos.module.scss";
+
+export const Kenos = () => {
+    //TODO: Add the useSelector and create the porcentage with the total, warningRange
+    let leftPosition = "80%";
+
+    const positionKenos = {
```

Comment:
Instead of creating this, you can use `leftPosition` directly in the div like this: `style={{ left: leftPosition }}`